searchcore/src/vespa/searchcore/proton/server/searchable_doc_subdb_configurer.cpp

```diff
@@ -87,6 +88,7 @@ void
 SearchableDocSubDBConfigurer::reconfigureSearchView(MatchView::SP matchView)
 {
     SearchView::SP curr = _searchView.get();
+    vespalib::eval::CompileCache::wait_pending();
```

Comment:
A comment here would be nice as well, as this relates to the matchers created earlier.

Reply:
https://github.com/vespa-engine/vespa/pull/13318